README.md

```diff
@@ -82,6 +82,6 @@ cargo run
 
 ## Documentation
 
-This follows the specification available in the [`api.asc`](https://github.com/solana-labs/ledger-app-solana/blob/master/doc/api.asc).
+This follows the [specification](https://github.com/solana-labs/ledger-app-solana/blob/master/doc/api.md).
```

Comment:
Can you rebase?  I fixed this in my last PR and made it a relative link, so that it follows any release branches.